tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "noEmit": true,
     "jsx": "react"
   },
-  "include": ["src"]
-}
\ No newline at end of file
+  "include": ["src", "src/setupTests.ts"]
```

Comment:
I think this one is redundant, because `src/` is already included. Correct me if I am wrong

Reply:
it was created automatically. I thought that was required to include it.